pool/web/view/component/component.ml

```diff
@@ -438,3 +453,31 @@ let selector
     ; div help
     ]
 ;;
+
+let custom_field_to_input ?flash_fetcher language custom_field =
+  let open Custom_field in
+  let label = Public.to_common_field language custom_field in
+  let help = Public.to_common_hint language custom_field in
+  let required = Public.get_required custom_field |> Required.value in
+  let create value input_type =
+    input_element
+      ?flash_fetcher
+      ?value
+      ?help
+      ~required
+      language
+      input_type
+      label
+  in
+  match custom_field with
+  | Public.Number field ->
+    let answer = field.Public.answer in
+    let value =
```

Comment:
- open `CCOption` at the beginning of the function
- switch input arguments of `create`

```suggestion
    field.Public.answer >|= (fun a -> a.Answer.value |> CCInt.to_string) |> create `Number
    (* and *)
    field.Public.answer >|= (fun a -> a.Answer.value) |> create `Text
```

or use a `Answer.to_string` function to make it even cleaner